recipes/poco/all/conanfile.py

```diff
@@ -66,15 +71,18 @@ class PocoConan(ConanFile):
     del comp
 
     generators = "cmake", "cmake_find_package"
-    _cmake = None
 
     @property
     def _source_subfolder(self):
-        return "source_subfolder"
+        return "src"
```

Comment:
why a new name?

Reply:
To decrease path length (32 chars for `build_subfolder/source_subfolder` against 9 for `build/src`, it's significant on windows), otherwise few build fail in CI due to recent modifications in Windows agents I guess. I want to avoid short_paths which is super tedious on Windows.

Reply:
it makes total sense. Thanks for answering!